src/main/java/com/psk/pyragas/ToolRent/usecases/Login.java

```diff
@@ -1,36 +1,43 @@
 package com.psk.pyragas.ToolRent.usecases;
 import com.psk.pyragas.ToolRent.dao.ProfilesDAO;
+import com.psk.pyragas.ToolRent.entities.Profile;
 import lombok.Getter;
 import lombok.Setter;
 
+import javax.enterprise.context.ApplicationScoped;
 import javax.enterprise.context.SessionScoped;
+import javax.inject.Inject;
 import javax.inject.Named;
 import java.io.Serializable;
+import java.util.List;
 
 @Getter
 @Setter
```

Comment:
If you have getters and setters on top of the class you don't need to repeat them for specific fields.